include/husky-base/mailbox_types.h

```diff
@@ -14,15 +14,20 @@
 
 #pragma once
 
+namespace husky {
+namespace base {
+
 // Mailbox event types
-const int MAILBOX_EVENT_RECV_COMM = 1;
-const int MAILBOX_EVENT_RECV_COMM_COMPLETE = 2;
-const int MAILBOX_EVENT_SET_RECV_COMM_HANDLER = 3;
-const int MAILBOX_EVENT_SET_RECV_COMM_COMPLETE_HANDLER = 4;
-const int MAILBOX_EVENT_SEND_COMM = 5;
-const int MAILBOX_EVENT_SEND_COMM_COMPLETE = 6;
-const int MAILBOX_EVENT_SHUTDOWN = 7;
+enum MailboxEventType : int32_t {
+  RecvComm = 1;
+  SetRecvHandler = 2;
+  SendComm = 3;
+  Shutdown = 4;
+}
```

Comment:
`};`?

Reply:
Should have flagged this PR as WIP...

But I'll merge it soon as I have another one depending on this.